### PR TITLE
About the index

### PR DESCRIPTION
**Pull request by fastfeee**
I don't if it's right ,but i have no other ideas

### CODE REVIEW

File: assets/index.html [hunk 1]
```diff
@@ -1,71 +1,405 @@
...
+            }else if (rid_value == 'b') {
+                const sendEncodings = [
+                    { rid: 'a', scaleResolutionDownBy: 2.0 }    
+                ]
+            }else{
+                const sendEncodings = [
+                    { rid: 'c' }    
+                ]
+            }
+            const stream = onPublish();
```
Comment (a-wing) [inline assets/index.html:349]: This function `onPublish` is not return any value

The stream is `undefined`

after:

Screenshot 2023-10-18 at 23 35 10

**Review by a-wing — CHANGES_REQUESTED** [review]
BTW: The `start Whip` is not working.

**Comment by fastfeee** [comment]
image
I dont't know why whe error occur

**Comment by fastfeee** [comment]
Is there a problem with the original index？
image

**Pull request closed without merging**